### PR TITLE
adding council page

### PR DESCRIPTION
**Pull request by rubyimrie**
adding council page, no big changes

### CODE REVIEW

File: components/MainNavBar.vue [hunk 1]
```diff
@@ -61,7 +61,7 @@
              { name : 'Named Witches', url: '/', external: false },
              { name : 'Detention Locations', url: '/', external: false },
              { name : 'Trial Locations', url: '/', external: false },
-             { name : 'Trials in Each Modern Authority', url: '/', external: false },
+             { name : 'Trials in Each Modern Authority', url: '/council', external: false },
```
Comment (Sephster) [inline components/MainNavBar.vue:64]: I think the link should be changed to be `trials-in-each-modern-authority` to match the link text

**Review by Sephster — CHANGES_REQUESTED** [review]
minor comment over the link but otherwise looks good. Thanks Ruby

**Review by Sephster — CHANGES_REQUESTED** [review]

File: components/MainNavBar.vue [hunk 2]
```diff
@@ -51,11 +51,13 @@
             :subpages="link.subpages"
           >
             {{ link.name }}
+            <<<<<<< HEAD =======
```
[the remark this answers is not in the document]
Comment (rubyimrie) [reply components/MainNavBar.vue]: done

[the remark this answers is not in the document]
Comment (Sephster) [reply components/MainNavBar.vue]: Thank you

**Review by Sephster — APPROVED** [review]

**Pull request merged**